src/controllers/userController.ts

```diff
@@ -0,0 +1,69 @@
+import { prisma } from "../model/prisma";
+import { Body, Set } from "../utils/utils";
+
+export const user = {
+  getById: async (userId: number, set: Set) => {
+    const user = await prisma.user.findUnique({
+      where: { id: userId },
+    });
+
+    if (!user) {
+      set.status = 400;
```

Comment:
Let’s create a custom `DB Error` which you will throw on any DP exceptions. You can handle them in the route with `.onError`

Reply:
Error code should be `404`